<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QDialog" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Customer{
border-image:url(:/Image/登录界面.png);

}



</string>
  </property>
  <widget class="QPushButton" name="ensure_order_pb">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>确认下单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_pb">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QListWidget" name="menu_listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>320</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(244,244,244,100);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>菜名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>单价(¥)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ordered_listWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(244,244,244,100);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>菜名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>小计(¥)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settle_account_pbt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>结账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_food_pbt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>催菜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_water_pbt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>加水</string>
   </property>
  </widget>
  <widget class="QPushButton" name="table_query_pbt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>265</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>餐桌状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="evaluate_pbt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>评价</string>
   </property>
  </widget>
  <widget class="QListWidget" name="progress_listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>320</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(244,244,244,100);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QLabel" name="current_table_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>265</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>265</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>当前桌位：</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>271</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>405</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>总计：¥</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>405</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>215</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>当前服务员ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="waiterid_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>215</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listView_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(254, 254, 254, 50);
border-radius:0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>评价星级</string>
   </property>
  </widget>
  <widget class="QLabel" name="dish_name_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>菜名</string>
   </property>
  </widget>
  <widget class="QLabel" name="dish_num_label">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ensure_order_pb</tabstop>
  <tabstop>quit_pb</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quit_pb</sender>
   <signal>clicked()</signal>
   <receiver>Customer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
